sled-agent/src/services.rs

```diff
@@ -267,6 +270,23 @@ impl ServiceManager {
             let smf_name = format!("svc:/system/illumos/{}", service.name);
             let default_smf_name = format!("{}:default", smf_name);
 
+            // Ensure the IPv6 traffic can be routed to this non-global zone.
+            //
+            // This is particularly important for accessing Nexus' external
+            // interface from off-device.
+            running_zone
+                .run_cmd(&["/usr/sbin/routeadm", "-e", "ipv6-forwarding"])
```

Comment:
Does this invocation not need a `-u` flag?

Reply:
I figured that was unnecessary because we're doing "-u" in the next invocation.

(I tried to do it in one line - `routeadm -e ipv6-forwarding ipv6-routing -u` - and the command seemed to be sad that I used multiple options at once)


... that being said, "-e option -e option" seems to work, so I'll do that.